roles/cloud_connector/tasks/delete_source.yaml

```
@@ -0,0 +1,23 @@
+---
+- name: Get source record
+  ansible.builtin.uri:
+    url: "https://{{ foreman_cloud_connector_console_redhat_host }}/api/sources/v3.1/sources?source_ref={{ satellite_uuid_json.json.results[0].value }}"
```

The filtering params here just need to be updated,
```suggestion
    url: "https://{{ foreman_cloud_connector_console_redhat_host }}/api/sources/v3.1/sources?filter[source_ref]={{ satellite_uuid_json.json.results[0].value }}"
```